src/Expr.js

```diff
@@ -128,6 +128,33 @@ var exprToString = function(expr, caller) {
     return varArgsFunctions.indexOf(caller) != -1 ? array : '[' + array + ']'
   }
 
+  if ('match' in expr) {
+    var matchStr = exprToString(expr['match'])
+    var terms = expr['terms'] || []
+
+    if (isExpr(terms)) terms = terms.raw
+
+    if (Array.isArray(terms) && terms.length == 0)
+      return 'Match(' + matchStr + ')'
+
+    if (Array.isArray(terms)) {
+      return 'Match(' + matchStr + ', ' + printArray(terms, exprToString) + ')'
+    }
```

Comment:
I added a simple verification to see if the terms is an array before call printArray. Sometimes, terms is not an array when the Match is called using one argument like `Match(Index(), "some")`.

Reply:
you should add right braces around `printArray` function because it doesn't print it.
```
if (Array.isArray(terms)) {
       return 'Match(' + matchStr + ', [' + printArray(terms, exprToString) + '])'
     }
```

Reply:
Usually, I think nested guard clauses are hard to read, what do you think? I also put the `[]` around the `printArray`.